### PR TITLE
Featured FAQs on homepage

### PR DESCRIPTION
**Pull request by jakearchibald**
@argyleink I've deviated with the design slightly here. Rather than give a preview of the FAQ item, I've gone with the whole thing, since they're pretty short. We can revisit this when we get the real content.

### CODE REVIEW

**Comment by argyleink** [comment]
cool, i'm down with this as a place to revisit, we'll solve the big text blocks soon. i'm hoping we can progressively enhance with some css line clamp, thoughts on that? might be minimal enough LOE to just nip that bud right now?

the deviations around color; I'd like those changed to match the comps. Gray cards on white is not as appealing as white cards on light gray. 

the typography is fine, but is uncanny, been trying to put my finger on it. the font sizes are great, i think it's the weights that need adjusted. the title is too bold, and the body copy should be set to thin. here's some quick changes i made with visbug to show what i mean

Screen Shot 2019-07-30 at 11 12 59 AM

i think once the weights change the blocks get truncated we'll be back to the less heavy feel. otherwise nice work, it's great getting content in there to work with!

**Comment by jakearchibald** [comment]
In terms of the light grey thing: it seems like modules alternate in colour, so I made that happen here. If we add a module before this, it'll appear more like the design. Or, do you want the colours fixed?

**Comment by argyleink** [comment]
i'd imagined colors were more fixed 👍

**Comment by jakearchibald** [comment]
Interesting, so the intent *isn't* that they alternate, it just coincidentally ended up that way in the XD?

**Comment by jakearchibald** [comment]
I guess they don't fully alternate. Ok, I'll fix this for all modules.

**Comment by jakearchibald** [comment]
I've done it all except for the line-clamping. I'd rather see the real content before we line clamp, and I'd rather we didn't have to line-clamp.

Personally, I find line-clamping annoying, as it's obscuring the information. If we have some FAQs that are short enough to feature, I'd rather we just featured the whole thing.

If we get to the point of needing clamping, I'd rather remove the content all together and featuring headings that link to the content.

**Comment by jakearchibald** [comment]
- [ ] Add iOS scrolling
- [ ] Add webkit prefix to max-content

**Comment by argyleink** [comment]
cool, i think i'm seeing a compromise / enhancement here based on your valid comments and the IA i intended for the home page. here's my thought process:
- i have a few FGs that we have a meaningful FAQ landing page
- i have a few FGs around the homepage IA pattern of collection summaries
- i agree line clamp has the effect of content masking, and i'm willing to design around that even though i dont fully agree it's always bad. low FGs

compromise idea: the FAQ summary/teaser module only shows FAQs that are within a certain answer text length. 
- no FAQ would be cut off
- no FAQ would be ginormous and make me want to clamp it
- still fits within the collection IA of being an intro to the greater FAQ list
- seems like a fun js function to write

thoughts!? 

also, i really like css snap points on horizontally scrolling lists like this and the media list. do you want me to add that requirement/request here, or make a new ticket for adding snap points to various horizontal scrollviews?

**Comment by jakearchibald** [comment]
> compromise idea: the FAQ summary/teaser module only shows FAQs that are within a certain answer text length.

I mean, that isn't a compromise idea, that's exactly what I've proposed and done.

> seems like a fun js function to write

In the current implementation you get to choose which FAQs to feature on the home page. This means you can, during PR review, assess whether they're the right size. No JS needed.

> also, i really like css snap points on horizontally scrolling lists like this and the media list.

I don't get it on the list of images. Like I said in the other ticket, I can't figure out the second half of "Scroll snap on a set of images helps the user because…"

IMO scroll-snap makes sense when you want one item to have focus, but that isn't true in these cases. I worry that it'd feel like we're dicking around with scrolling because it's a new shiny feature.

File: .eleventy.js [hunk 1]
```diff
@@ -94,5 +100,14 @@ module.exports = function(eleventyConfig) {
     return sections;
   });
 
+  eleventyConfig.addCollection('featuredFAQs', collection => {
+    const faqs = collection
+      .getFilteredByTag('faq')
+      .filter(item => item.data.featured)
```
Comment (argyleink) [inline .eleventy.js]: 👍

**Comment by argyleink** [comment]
coo, so with a featured flag, my guidance would be to select featured FAQs based on length and relevance. we want short and meaningful FAQs in the featured area 👍 

curious if these guidances will be enforceable at the CMS level, or at least blurbs we can share. that FAQ module space is easy to make ugly. but these worries are also very future facing for the white labeled version. i'll pacify them for now!

scrollsnap points is part of the design btw, def not just using it for fun, i want that left alignment as you scroll. it can be so subtle folks dont even realize it's there, and if the visiting browser doesnt support it, they'll get a great experience anyway. i'd prefer we had snapping on most of these homepage collection scrollviews, but maybe that's another ticket to make so it doesnt hinder the current PR?

**Comment by jakearchibald** [comment]
Yeah, let's make the scrollsnap thing another PR. Unfortunately the implementation isn't as trivial as it might seem.

**Pull request merged**